ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -37,7 +37,20 @@ export class EvalCommand extends Command<EvalCommandReturn> {
 
   private evalFromConfigsOverride(result: EvalCommandReturn): EvalCommandReturn {
     if (!this.parameters.configs) {
-      return result;
+      return _.mapValues(result, (current) => {
```

Comment:
The condition here is incorrect, if you have two lazy keys and one is provided and the second isn't you'll end up evaluating from the store for the missing key.

How about this direction? We let the existing override logic always run and check if there is a lazy property on the keys
```node.js
  private evalFromConfigsOverride(result: EvalCommandReturn): EvalCommandReturn {
    return _.mapValues(result, (current) => {
      const { context } = current;
      const hasConfigOverrideValue = Object.prototype.hasOwnProperty.call(this.parameters.configs, context.key);

      if (!hasConfigOverrideValue && !context.cfgu.lazy) {
        return current;
      }

      const configOverrideValue = this.parameters.configs?.[context.key] ?? '';
      return {
        ...current,
        result: {
          origin: EvaluatedConfigOrigin.ConfigsOverride,
          source: `parameters.configs.${context.key}=${configOverrideValue}`,
          value: configOverrideValue,
        },
      };
    });
  }
```

Reply:
done

Reply:
done